src/components/tutorialCard/tutorialCard.js

```diff
@@ -26,7 +26,7 @@ const TutorialCard = props => (
     }}
   >
     <CardHeader>
-      <h2 className="pf-c-title pf-m-lg">{props.title}</h2>
+      <h2 className="pf-c-title pf-m-xl">{props.title}</h2>
```

Comment:
Can you also change this to a <h3> instead?

Reply:
Good catch... done and tested ok. I'll put it on my server as well.